Dockerfile-dcrd

```diff
@@ -4,43 +4,63 @@
 
 FROM ubuntu:xenial
 
-LABEL description="Decred mainnet daemon"
-LABEL version="1.1"
-LABEL maintainer "john@netpurgatory.com"
+LABEL description="Decred mainnet daemon" \
```

Comment:
This change makes no difference in the final image size so please put it back to the way it was.  I think the one LABEL per line is clearer.